docker-compose/technical/docker-compose.technical.yml

```
@@ -45,6 +45,17 @@ services:
       - $PWD/../technical/init-merging-data.sh:/init-merging-data.sh:Z
     restart: unless-stopped
 
+  postgres-exporter:
+    profiles:
+      - all
+      - metrics
+    image: prometheuscommunity/postgres-exporter:v0.15.0
+    ports:
+      - "9187:9187"
+    environment:
+      - DATA_SOURCE_NAME=postgresql://postgres:postgres@postgres:5432/postgres?sslmode=disable
```

in [the documentation](https://github.com/prometheus-community/postgres_exporter#environment-variables), it say
> DATA_SOURCE_NAME the default legacy format. Accepts URI form and key=value form arguments. The URI may contain the username and password to connect with.

Wouldn't it be better not to use legacy parameters even if it's the default?
```suggestion
      DATA_SOURCE_USER: postgres
      DATA_SOURCE_PASS: postgres
      DATA_SOURCE_URI: postgres:5432/postgres?sslmode=disable
```